src/libcore/lib.rs

```diff
@@ -62,6 +62,9 @@
 #![warn(missing_docs)]
 #![warn(intra_doc_link_resolution_failure)]
 #![warn(missing_debug_implementations)]
+#![deny(rust_2018_idioms)]
```

Comment:
For consistency with the other crates, could you add an empty line above this?

(Yes I know I have an issue if I care about this.^^)